.travis.yml

```diff
@@ -17,11 +18,12 @@ cache:
     - $HOME/.composer/cache
 
 before_script:
-  - travis_retry composer install -n
+  - travis_retry composer self-update
+  - travis_retry composer install --no-interaction --prefer-source
```

Comment:
Why prefer source?

Reply:
travis will download current sources directly from git without any questions

Reply:
I'm not asking what it does, I'm asking why are you changing the build to do that? What advantages does it provide?

Every pull request should explain the changes it contains. Don't expect me to guess.